README.md

```diff
@@ -3,12 +3,12 @@
 # ponyo 
 [![Coverage Status](https://coveralls.io/repos/github/greenelab/ponyo/badge.svg?branch=master)](https://coveralls.io/github/greenelab/ponyo?branch=master)
 
-**Alexandra Lee and Casey Greene 2020**
+**Alexandra J. Lee, Ben Heil and Casey S. Greene 2020**
```

Comment:
I don't know that my contributions warrant being listed at the top of the repo since I just refactored some functions. I should probably be down in the acknowledgements with Marvin

Reply:
Sure. Its whatever you're comfortable with. I can make that change.